sdk/tests/integration/ClientIntegrationTest.cc

```diff
@@ -47,22 +61,14 @@ class ClientIntegrationTest : public ::testing::Test
   [[nodiscard]] inline const AccountId& getAccountId() const { return mAccountId; }
   [[nodiscard]] inline const std::string getPathToJSON() const { return mFilePath.string(); }
 
-  [[nodiscard]] inline const std::chrono::milliseconds getNegativeBackoffTime() const { return mNegativeBackoffTime; }
-  [[nodiscard]] inline const std::chrono::milliseconds getZeroBackoffTime() const { return mZeroBackoffTime; }
-  [[nodiscard]] inline const std::chrono::milliseconds getBelowMinBackoffTime() const { return mBelowMinBackoffTime; }
-  [[nodiscard]] inline const std::chrono::milliseconds getAboveMaxBackoffTime() const { return mAboveMaxBackoffTime; }
-
 private:
+  // Client mClient;
```

Comment:
Should get rid of this commented-out code.

Reply:
Removed in b8259a713eabcbeb554889a5e5e9a602f4690bac